### ISSUE TITLE
Win 11 test

### ISSUE THREAD

**Issue by natanaelmgomes** [brex97/BLE_TEST_PythonApp#1]
Have you tested this app in Windows 11?
I cannot read a service and I suspect it can be related to OS problems.

**Comment by muschter** [comment]
@natanaelmgomes You are correct. I tested this app on Windows 11 and could not read services/characteristics, then tested it on a Windows 10 computer and everything worked fine.

**Comment by sztangwang** [comment]
I can't even use windows 10
